version_mapping/3.1.0-4.0.0/upgrade-shapes-310-to-400.ttl

```diff
@@ -151,7 +151,7 @@ ids:ResourceUpgradeShape
 						
 			OPTIONAL {
 				$staticEndpoint ids:endpointHost $endpointHost .
-				$this rdf:type ids:Host .
+				$this rdf:type ids:Endpoint .
```

Comment:
```suggestion
				$this rdf:type ids:Host .
```

It was actually Host in this previous version, so you must not change that here. Please revert, thank you.